src/somef/utils/constants.py

```diff
@@ -120,7 +120,12 @@
 CAT_WORKFLOWS = "workflows"
 CAT_TYPE = "type"
 CAT_CONTINUOS_INTEGRATION= "continuousIntegration"
-
+########### CATEGORIES RELATED TO PACKAGES ##########
+CAT_PACKAGE_FILE = "package_file"
```

Comment:
Why is there a metadata category for package files? It does not make sense to me
Categories related to packages should be included in the ones we already have.